src/tokens.scss

```
@@ -0,0 +1,9 @@
+:host {
+  --ds-auro-toast-background-color: var(--ds-color-brand-gray-400, $ds-color-brand-gray-400);
+  --ds-auro-toast-border-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  --ds-auro-toast-close-button-icon-color: var(--ds-color-text-primary-inverse, $ds-color-text-primary-inverse);
+  --ds-auro-toast-close-button-background-color: transparent;
+  --ds-auro-toast-close-button-border-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  --ds-auro-toast-icon-color: var(--ds-color-text-primary-inverse, $ds-color-text-primary-inverse);
+  --ds-auro-toast-text-color: var(--ds-color-text-primary-inverse, $ds-color-text-primary-inverse);
+}
```

Two things. 

1. When using Sass variables, they need to be imported into the document
2. When using Sass variables in this way, you need to use [Sass Interpolation](https://sass-lang.com/documentation/interpolation/)

```suggestion
// Import Auro tokens
@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';

:host {
  --ds-auro-toast-background-color: var(--ds-color-brand-gray-400, #{$ds-color-brand-gray-400});
  --ds-auro-toast-border-color: var(--ds-color-border-divider-default, #{$ds-color-border-divider-default});
  --ds-auro-toast-close-button-icon-color: var(--ds-color-text-primary-inverse, #{$ds-color-text-primary-inverse});
  --ds-auro-toast-close-button-background-color: transparent;
  --ds-auro-toast-close-button-border-color: var(--ds-color-border-divider-default, #{$ds-color-border-divider-default});
  --ds-auro-toast-icon-color: var(--ds-color-text-primary-inverse, #{$ds-color-text-primary-inverse});
  --ds-auro-toast-text-color: var(--ds-color-text-primary-inverse, #{$ds-color-text-primary-inverse});
}
```